source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+# Cluster
+
+## The ClusterState Resource
+
+<aside class="notice">
+This section describes API features that are part of the Cloudify premium edition
+</aside>
+
+```python
+# include this code when using cloudify python client-
```

Comment:
~include this code when using cloudify python client-~ **Include this code when using the Cloudify Python client-**